conans/model/build_info.py

```diff
@@ -190,6 +198,7 @@ def _raise_if_mixing_components(self):
                 self.libdirs != [DEFAULT_LIB] or
                 self.bindirs != [DEFAULT_BIN] or
                 self.resdirs != [DEFAULT_RES] or
+                self.builddirs != [DEFAULT_BUILD] or
```

Comment:
is this something for a different PR to be merged asap?

Reply:
I don't think it is very relevant as it is something experimental